shared/utils/handleError.ts

```diff
@@ -0,0 +1,8 @@
+const handleError = (e: Error) => {
+  // any error handling/logging here
+  // eslint-disable-next-line no-console
+  console.error(e);
+  alert(e); // killer feature %)
```

Comment:
i guess it shouldn't be in `shared` since it fails in node js 🙂

Reply:
:D
moved to src folder.